### PR TITLE
Properly handle multiple use of pac4j

### PR DESCRIPTION
**Pull request by leleuj**
```
master: https://github.com/apereo/cas/pull/5796
```

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -255,7 +255,7 @@ commonsCollectionsVersion=4.4
 # Pac4j versions
 ###############################
 pac4jSpringWebmvcVersion=6.0.3
-pac4jVersion=5.4.6
+pac4jVersion=5.7.2-SNAPSHOT
```
Comment (mmoayyed) [inline gradle.properties:258]: This cannot happen. I realize you likely need this for `val sessionStore = new JEESessionStore();`. Instead, define a subclass for JEESessionStore and use that instead. Keep the pac4j version as is.

Comment (mmoayyed) [reply gradle.properties:258]: Yes that is correct. Doing this violates the release policy, and possibly introduces (may) breaking changes for others.

Comment (leleuj) [reply gradle.properties]: It perfectly makes sense. I shouldn't have forgotten

**Comment by leleuj** [comment]
@mmoayyed I have completed a Puppeteer test to ensure the logout (for a delegated authn) works (even if the OIDC server support is used) and duplicated the same test with a no-replication configuration.
To be sure it works on both cases (`PrefixedSessionStore` and `DistributedJEESessionStore`).

Done on the master too.

**Comment by leleuj** [comment]
You're welcome

**Pull request merged**